cabal-install/Distribution/Client/VCS.hs

```diff
@@ -498,3 +499,84 @@ svnProgram = (simpleProgram "svn") {
         _ -> ""
   }
 
+
+-- | VCS driver for Pijul.
+-- Documentation for Pijul can be found at <https://pijul.org/manual/introduction.html>
+--
+vcsPijul :: VCS Program
+vcsPijul =
+    VCS {
+      vcsRepoType = KnownRepoType Pijul,
+      vcsProgram  = pijulProgram,
+      vcsCloneRepo,
+      vcsSyncRepos
+    }
+  where
+    vcsCloneRepo :: Verbosity -- ^ it seams that pijul does not have verbose flag
```

Comment:
I'm quite sure it does, let us find out.

Reply:
... but indeed, it looks like there are no flags to affect output of `pijul` atm. Surprising :)